src/main/pages/setup-kubernetes/kubernetes-multi-user.adoc

```diff
@@ -1,80 +1,87 @@
+.. Bind it to the almighty `cluster-admin` role: `kubectl apply -f ./tiller-rbac.yaml`.
+.. Install `tiller`: `helm init --service-account tiller`.
+* Ensure that you have an NGINX-based Ingress controller. This is the default Ingress controller on Minikube. To start it, use the `minikube addons enable ingress` command.
+* DNS discovery should be enabled. It is enabled by default in Minikube.
 
 [id="cluster-ip"]
-== Cluster IP
+== Obtaining the cluster IP
 
-* If your cluster is running on MiniKube, simply type `minikube ip` at your terminal
-* If your cluster is in the cloud, obtain the hostname or ip address from your cloud provider
+To obtain the cluster IP:
 
-In production, you should specify a hostname (see https://github.com/eclipse/che/issues/8694[here] why). In case you don’t have a hostname (e.g. during development), and would still want to use a host-based configuration, you can use services such as nip.io or xip.io.
+* If your cluster is running on Minikube, type `minikube ip` on the command line.
+* If your cluster is in the cloud, obtain the hostname or IP address from your cloud provider.
 
-In case you’re specifying a hostname, simply pass it as the value of the `ingressDomain` parameter below.
+In production, you must specify a hostname. For details, see https://github.com/eclipse/che/issues/8694[Add an ability to use host-based routing instead of path-based for k8s infras]. If you do not have a hostname and want to use a host-based configuration, use services such as nip.io or xip.io.
 
-If you must use IP address (e.g. your corporate policy prevents you from using nip.io), you would also have to set `isHostBased` to `false`.
+If specifying a hostname, pass it as the value of the `ingressDomain` parameter in the deployment syntax in the following section.
+
+If using the IP address, for example, your corporate policy prevents you from using nip.io, you must also set `isHostBased` to `false`.
 
 [id="deploy-syntax"]
-== Deploy Syntax
+== Deployment Syntax
 
-The context of the commands below is `che/deploy/kubernetes/helm/che`
+The context of the commands below is `che/deploy/kubernetes/helm/che`.
 
 [id="to-deploy-with-dedicated-keycloak-as-authentication-server"]
-=== To deploy with dedicated keycloak as authentication server:
+=== Deploying with dedicated Keycloak as authentication server
+
+To deploy multi-user with dedicated Keycloak as authentication server:
 
-* Override default values by changing the values.yaml file and then typing:
+. Override default values by changing the `values.yaml` file. On the command line, run the following command:
 +
 ----
 helm upgrade --install <my-che-installation> --namespace <my-che-namespace> -f ./values/multi-user.yaml ./
 ----
-* Or, you can override default values during installation, using the `--set` flag:
++
+Or, override default values during installation, using the `--set` flag:
```

Comment:
do we need two ways? choose the better/simpler way and go with it. if both really need to be documented, lead in with s/t like:

"The following two methods can be used to override..."